### PR TITLE
[MB-15173] Incorporate weights moved/weights moved by customer shipment list components to display shipment level weights

### PR DESCRIPTION
**Pull request by afoon**
## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the UI locally.

```sh
make client_run
```

##### Terminal 2

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Access the Office app as service counselor.
2. Navigate to `counseling/moves/XSWT05/review-shipment-weights/`
3. Shipment tables should be visible

## Screenshots
Screenshot 2023-03-14 at 3 53 38 PM

### CODE REVIEW

**Review by AnnaGingle — CHANGES_REQUESTED** [review]
The shipment rows are highlighted blue on hover. This didn't happen in storybook. This gives the user the impression the shipment row is a button.

**Review by AnnaGingle — APPROVED** [review]
LGTM!

**Review by michaelvaldes-truss — APPROVED** [review]
Looks great! 🚀

File: src/components/Office/PPM/ReviewShipmentWeightsTable/ReviewShipmentWeightsTable.module.scss [hunk 1]
```diff
@@ -8,11 +8,20 @@
   flex-direction: column;
   flex: auto;
 
+  .table{
+    min-height: 0;
+    tr:hover {
+      background-color: white;
```
Comment (michaelvaldes-truss) [inline src/components/Office/PPM/ReviewShipmentWeightsTable/ReviewShipmentWeightsTable.module.scss:14]: **suggestion:** I think the linter is complaining about an extra space at the end of this line -- you might wanna just reformat the whole file in your editor though

File: src/components/Office/PPM/ReviewShipmentWeightsTable/ReviewShipmentWeightsTable.module.scss [hunk 2]
```diff
@@ -8,11 +8,20 @@
...
   flex: auto;
 
+  .table{
+    min-height: 0;
+    tr:hover {
+      background-color: white; 
+    }
+  }
+
+
```
Comment (michaelvaldes-truss) [inline src/components/Office/PPM/ReviewShipmentWeightsTable/ReviewShipmentWeightsTable.module.scss:18]: **suggestion (non-blocking):** you can remove the extra new line here

**Pull request merged**